### DISCUSSION TITLE
No Tuning Tool - what about O&O Shutup10 and such

### DISCUSSION THREAD

**Discussion by apollo7321** [beerisgood/Windows11_Hardening#24] [Q&A] [answered]
Hello,

the list says no tuning tools but no examples. My first thought were stuff like CCleaner, but what about tools that change group policies and such like O&O Shutup10 and Winaero Tweaker?

**Addition:** While reading I got another question, but I did't want o create a new discussion. What about "Use a Microsoft account" for home users? What is that good for?

Thanks

**Comment by beerisgood** [comment]
I wrote 
> No "Tuning" tools (not even stuff like Ccleaner!)

If I’m not wrong O&O Shutup10 did something bad in the past. Didn’t they even collect user data?
So yeah I’m against all of that.

> Use a Microsoft account

where did you read this?

**Reply by apollo7321** [reply]
Oh, did they? I haven't heard of it. So far it was the easiest way to change group policies on Win 11 Home. I'll look into it.

> where did you read this?

here: https://github.com/beerisgood/Windows11_Hardening/blob/master/Microsoft%20recommendations%20for%20the%20average%20home%20use#L8

**Reply by beerisgood** [reply, marked as answer]
Ah yeah this is from a forum guy. 

well with an Microsoft account you can sync your windows data but the most important thing is Windows Hello which is the most secure login feature for windows.

**Reply by apollo7321** [reply]
I see. Thank you for your quick responses.
